app/konnect/gateway-manager/data-plane-nodes/index.md

```diff
@@ -36,7 +36,7 @@ Advanced setup:
 {:.note}
 > **Notes:** 
 > * Gateway Manager includes a feature called Control Plane Launcher which can be used with any of AWS, Azure, or GCP. This feature is currently in tech preview.
-> * Kong does not host data plane nodes. You must install and host your own.
+> * Kong only hosts fully-managed Dedicated Cloud Gateway data plane nodes. You must install and host your own if you want to use a self-managed data plane.
```

Comment:
We host both control planes and data planes now so the word 'only' here throws me off. Maybe "Kong can host data planes with our Dedicated Cloud Gateways offering, if you want to use a self-managed data plane, you must and install and host your own"

Reply:
I think we can remove this note altogether. CGWs _are_ the hosted offering.